src/Sulu/Component/Content/Metadata/Loader/StructureXmlLoader.php

```diff
@@ -71,13 +70,14 @@ class StructureXmlLoader extends AbstractLoader
     ];
 
     /**
-     * tags that are required in template
-     * TODO should be possible to inject from config.
+     * Properties that are required in template.
      *
      * @var array
      */
     private $requiredPropertyNames = [
-        'title',
+        'page' => ['title'],
+        'home' => ['title'],
+        'snippet' => ['title'],
```

Comment:
Shouldn't that be gone all together? These values will never be used, because they are always overridden in the constructor, right?

Reply:
:+1: